meson.build

```diff
@@ -103,6 +103,12 @@ else
   vst_cpp_args = ['-DDEVELOPMENT']
 endif
 
+if cxx.has_function('gst_get_main_executable_path',
+                  prefix: '#include <gst/gst.h>',
+                  dependencies: gst_dep)
+  vst_cpp_args += ['-DHAVE_GST_EXE_PATH']
+endif
```

Comment:
That works too, I would've just `#if GST_CHECK_VERSION(1,13,0)` but this here is of course more accurate :)

Reply:
It was so easy I figured I might as well :)